intellij-plugin/src/main/resources/messages/ApolloBundle.properties

```
@@ -79,8 +79,12 @@ CompatToOperationBasedCodegenMigrationProcessor.noUsage=No compat codegen usage
 
 SandboxService.OpenInSandboxAction.text=Open in Apollo Sandbox
 
-inspection.group.graphql.studio=Apollo GraphOS
 inspection.group.graphql=GraphQL
+inspection.group.graphql.studio=Apollo GraphOS
+inspection.group.graphql.apolloKotlin=Apollo Kotlin
 inspection.fieldInsights.displayName=Expensive field
 inspection.fieldInsights.reportText=<html><b><tt>{0}</tt></b> is expensive to resolve: <b>{1}</b>
 inspection.fieldInsights.settings.threshold=Threshold (ms)
+inspection.schemaInGraphqlFile.displayName=Schema in .graphql file
+inspection.schemaInGraphqlFile.reportText=<html>Schema definitions must reside in a <tt>.graphqls</tt> file
```

Nit. Make it clearer that it's an `Apollo Compiler` thing (vs a more general GraphQL thing)?

```suggestion
inspection.schemaInGraphqlFile.reportText=<html>The Apollo Kotlin compiler requires type definitions to reside in a <tt>.graphqls</tt> file
```